plugins/inputs/p4runtime/p4runtime.go

```diff
@@ -11,14 +11,14 @@ import (
 	"slices"
 	"sync"
 
-	p4ConfigV1 "github.com/p4lang/p4runtime/go/p4/config/v1"
-	p4v1 "github.com/p4lang/p4runtime/go/p4/v1"
+	p4_config_v1 "github.com/p4lang/p4runtime/go/p4/config/v1"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
```

Comment:
Are the v1 suffixes needed?
```suggestion
	p4_config "github.com/p4lang/p4runtime/go/p4/config/v1"
	p4 "github.com/p4lang/p4runtime/go/p4/v1"
```

Reply:
No, they aren't needed. Changed.